.changeset/twenty-onions-fetch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/source-foundations': patch
```

Comment:
Would this not be a feature? If so, it’s `minor`.

Reply:
i think it's more a internal refactor? the api shouldn't change (iiuc)

Reply:
i'll do a final check after we've resolved the other discussion points to make sure the api doesn't change

Reply:
the e2e tests should catch anything too - as long as `[source-foundations/src/index.ts](https://github.com/guardian/source/pull/1500/files#diff-4d48fdc382593a97a362bf40e2b2156e09271531b8776cb413d957fd1f422d10)` doesn't have any changes, should be good